client/src/js/components/bhSoldOut/bhSoldOut.html

```diff
@@ -0,0 +1,36 @@
+<table
+ng-show = "$ctrl.soldOutInventories.length > 0"
+class="table table-condensed table-bordered">
+<thead>
+  <tr>
+    <th>
+      <span class="text-danger" 
+        translate-values="{ total : $ctrl.soldOutInventories.length }"
+        translate="{{ 'STOCK.STOCK_OUT_WARNING' }}">
+      </span>
+    </th>
+  </tr>
+</thead>
+<tbody>
+  <tr ng-repeat = "inventory in $ctrl.soldOutInventories | limitTo : 5">
+    <td>{{inventory.text}} ({{inventory.stock_out_date}})</td>
+  </tr>
+
+  <tr>
+    <td>
+      <span class="text-danger" 
+        ng-show="($ctrl.soldOutInventories.length - 5) > 0"
+        translate-values="{ left : ($ctrl.soldOutInventories.length - 5)}"
+        translate="{{ 'STOCK.AND_MORE' }}">
+      </span>
+
+    <a ui-sref= "stockInventories({ filters : [
+      { key : 'period', value : 'allTime'},
+      { key : 'includeEmptyLot', value : 1  },
+      { key : 'depot_uuid', value : $ctrl.depot.uuid, displayValue: $ctrl.depot.text, cacheable:false },
+      { key : 'status', value : 'stock_out', displayValue: 'STOCK.STATUS.STOCK_OUT', cacheable:false }
+      ]})"translate>FORM.LABELS.SEE_ALL</a>
```

Comment:
It looks like this translate is stuck on the ".  Can you put a space between the two?